test/lib/usd/translators/testUsdExportUVSets.py

```diff
@@ -374,7 +363,8 @@ def testExportUvVersusUvIndexFromIterator(self):
         else:
             stPrimvar = brokenBoxMesh.GetPrimvar("map1").ComputeFlattened()
 
-        self.assertEqual(stPrimvar[0], Gf.Vec2f(1.0, 1.0))
+        self.assertEqual(stPrimvar[0], Gf.Vec2f(1.0, 2.0))
+
```

Comment:
This UV set was previously getting compressed to `vertex` interpolation. With the compression now removed, it remains as `faceVarying`, and the first UV in the flattened array is now a different value.